test/integration_test.sh

```diff
@@ -115,11 +118,12 @@ git clone git@github.com:HumanCellAtlas/mint-deployment.git
 mint_deployment_dir=mint-deployment
 
 # 2. Clone Lira if needed
-if [ $lira_mode == "github" ]; then
+if [ $lira_mode == "github" ] || [ $lira_mode == "image" ]; then
```

Comment:
We need a copy of the Lira repo no matter the mode so that the Lira config template is available.

Reply:
Would you mind mentioning that in the script comment?

Reply:
There is still a path in step 2 where lira_mode is "local", where we don't clone Lira. I will add a comment up near the top of the script to explain that we do clone in image mode but only to get the config template.